src/registrar/tests/test_management_scripts.py

```diff
@@ -33,7 +34,9 @@ def setUp(self):
         self.unknown_domain, _ = Domain.objects.get_or_create(name="fakeunknown.gov", state=Domain.State.UNKNOWN)
 
         # Set a ready_at date for testing purposes
-        self.ready_at_date = datetime.date(2022, 12, 31)
+        self.ready_at_date = date(2022, 12, 31)
+        _ready_at_datetime = datetime.combine(self.ready_at_date, time.min)
+        self.ready_at_date_tz_aware = timezone.make_aware(_ready_at_datetime, timezone=timezone.utc)
```

Comment:
(No action needed) I wonder why this wasn't being catched by with_less_console_noise... Maybe something to do with scope?

Reply:
I am not sure.  It is a good question.  It was good though not to have the noise; otherwise, these warnings were not being caught